src/api/spec/factories/configuration.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :configuration do
```

Comment:
Configuration factory isn't needed... We already excluded from the truncation of the tables, so we always have a Configuration in the RSpec tests